api/drivers/aws/driver.js

```diff
@@ -107,6 +109,74 @@ class AWSDriver extends Driver {
   }
 
   /**
+   * Get the aws price
+   * It reduce the AwsPrice Object by deleting useless key
```

Comment:
Can you precise this method's job is to set `this.awsPrice` to a promise that will resolve to the prices retrieved from AWS ?